scripts/core/program.yml

```diff
@@ -10,29 +10,47 @@ type: header
 desc: "Intel $OneApi Level-Zero Runtime APIs for Program"
 ordinal: "2"
 --- #--------------------------------------------------------------------------
+type: fptr_typedef
+desc: "callback function for urProgramCreate"
+name: $x_programcreate_callback_t
+return: void
+params:
+    - type: $x_program_handle_t
+      name: hProgram
+      desc: "[in] handle of Program object created."
+    - type: void*
+      name: pParams
+      desc: "[in][out] pointer to user data to be passed to callback."
+--- #--------------------------------------------------------------------------
 type: function
 desc: "Create Program from input SPIR-V modules."
 class: $xProgram
 name: Create
 decl: static
 ordinal: "0"
 analogue:
-    - "**clCreateProgram**"
+    - "**clCreateProgramWithIL** and **clBuildProgram**"
```

Comment:
I feel like this is closer to `clCompileProgram` since `clBuildProgram` does an implicit link.

Reply:
This comment also applies to the `ERROR_PROGRAM_BUILD_FAILURE` error code.